install_config/aggregate_logging.adoc

```diff
@@ -855,7 +855,18 @@ spec:
 ----
 <1> This label should uniquely identify a replica with a single node that bears that
 label, in this case `logging-es-node=1`. Use the `oc label` command to apply
-labels to nodes as needed.
+labels to as many nodes as needed.
+
+For example, if your deployment has three infrastructures, you could add labels for those
+infrastructures as follows:
+----
+oc label node <nodename1> logging-es-node=1
+oc label node <nodename2> logging-es-node=2
+oc label node <nodename3> logging-es-node=3
+----
+
+For information about adding a label to an infrastructure, see
+xref:../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Updating Labels on Nodes].
 
 To automate applying the node selector you can instead use the `oc patch` command:
 
```

Comment:
In line 874, what's the suffix? I'm also not sure if that command is mean to replace both editing the DC and applying the label to the node or just editing the DC. (I think it's just editing the DC, so it might need to be an option for the first step.)

Remove the ==== at line 877 and 846

Reply:
Thanks.  removed ==== at those lines.

Checking with SMEs about the suffix.